teams/workstations.yml

```diff
@@ -26,21 +27,15 @@ controls:
       macos_setup_assistant: ../lib/automatic-enrollment.dep.json
   scripts:
   - path: ../lib/remove-zoom-artifacts.script.sh
-  - path: ../lib/reset-timezone.sh
+  - path: ../lib/set-timezone.script.sh
 queries: 
 - path: ../lib/collect-usb-devices.queries.yml
 - path: ../lib/collect-failed-login-attempts.queries.yml
-policies: 
-- path: ../lib/macos-device-health.policies.yml
-- path: ../lib/windows-device-health.policies.yml
-- path: ../lib/windows-device-health.policies.yml
-agent_options: ./lib/agent-options.yml
+policies:
+  - path: ../lib/macos-device-health.policies.yml
+  - path: ../lib/windows-device-health.policies.yml
+agent_options:
+  path: ../lib/agent-options.yml
 team_settings:
-  webhook_settings:
-    failing_policies_webhook:
-        enable_failing_policies_webhook: true
-        destination_url: https://example.tines.com/webhook
-        policy_ids: [1, 2, 3, 4, 5,6 ,7, 8, 9]
```

Comment:
Are we pulling these out to simplify the example? Or, did we cut `webhook_settings` functionality?

Reply:
Our current `fleetctl apply` does not support team webhook_settings or integrations, right? And I'm using the existing APIs, so `fleetctl gitops` doesn't support these either without additional work.

Reply:
> `fleetctl apply` does not support team `webhook_settings` or `integrations`, right?

@getvictor I'm not positive.

@mna when you get the chance, can you please sanity check us here?

If this is the case, it makes sense to punt on adding support for `webhook_settings` and `integrations`. We can add support for this in `fleetctl gitops` later.

Reply:
AFAICT that's correct, no webhook or integrations can be set via apply team specs: https://github.com/fleetdm/fleet/blob/main/server/fleet/teams.go#L387